sgtm/apollo/README.md

```diff
@@ -0,0 +1,196 @@
+# gPS Apollo
+
+*Get data from a Google Sheet in realtime to generate lead gen value
+for lead scoring. This value will be activated via VBB (value based bidding) to
+optimize Google Ads campaign performance*
+
+Use
+[Server Side Tag Manager](https://developers.google.com/tag-platform/tag-manager/server-side)
+with [Spreadsheet API](https://developers.google.com/sheets/api/guides/concepts), 
+to generate real time
+conversion value and use them with Google Ads products and other
+services to improve your targeting and bidding processes.
+
+> Tip 💡: Check out [the sGTM Pantheon](
+    https://github.com/google-marketing-solutions/gps-sgtm-pantheon/tree/main)
+for more tags for different use-cases.
+
+## Description
+
+![Description 1](docs/img/description_1.png "Description 1")
+
+We know that the client begins with the user clicking on the ad
+and arrives at the website then the user browses your site,
+reads about your product and services
+and then later fills in a form on the site and becomes a lead for the business.
+This will be stored in the CRM and database and then later when a lead is closed,
+this will be uploaded in Google Ad.
+
+So looking at the above workflow, 
+you can see that there are several cons here.
+First it can take several weeks or days for lead gen clients 
+to know the true value of conversion. 
+And we know that the longer time it takes to upload the closed lead information, 
+the less impact it would have on bidding.
+
+
+![Description 2](docs/img/description_2.png "Description 2")
+
+So our solutions improve or enhance the current standard of workflow, 
+if you look at the third step, we have two options.
+
+The first option is gPS-Apollo solution, [gPS-Apollo solution](https://github.com/JunghanYoon/gps-apollo). 
```

Comment:
This link is outdated. I'd consider removing this completely, as it's not needed.

Reply:
Yes, you're right. Thank you for catching it!
I removed this completely!

Thank you!